sycl/plugins/unified_runtime/ur/adapters/cuda/command_buffer.cpp

```diff
@@ -148,7 +183,7 @@ UR_APIEXPORT ur_result_t UR_APICALL urCommandBufferAppendKernelLaunchExp(
           cuGraphAddEmptyNode(&GraphNode, hCommandBuffer->CudaGraph,
                               DepsList.data(), DepsList.size()));
 
-      // Get sync point and register the event with it.
+      // Get sync point and register the cuNode with it.
       *pSyncPoint = hCommandBuffer->GetNextSyncPoint();
       hCommandBuffer->RegisterSyncPoint(
           *pSyncPoint, std::make_shared<CUgraphNode>(GraphNode));
```

Comment:
Does it make sense to add a helper routine for the sync point to not have to duplicate this code throughout?

Reply:
I added an helper to register sync points.